docs/source/configuration_file.md

```diff
@@ -0,0 +1,139 @@
+# Configuration file
+
+* [General parameters](#general-parameters)
+* [Loader parameters](#loader-parameters)
+* [Split dataset](#split-dataset)
+* [Training parameters](#training-parameters)
+* [Architecture](#architecture)
+* [Testing parameters](#testing-parameters)
+* [Transformations](#transformations)
+* [Examples](#examples)
+
+## General parameters
+#### command
+Run the specified command. Choices: ``"train"``, ``"test"``, ``"eval"``, to train, test and evaluate a model respectively.
+#### gpu
+Integer. ID of the GPU to use.
+#### log_directory
+Folder name where output files (e.g. trained model, predictions, results) are saved.
+#### debugging
+Bool. Allows extended verbosity and intermediate outputs.
+
+## Loader parameters
+#### bids_path
+String. Relative path of the BIDS folder.
+#### target_suffix
+Suffix list of the derivative file containing the ground-truth of interest (e.g. [`"_seg-manual"`, `"_lesion-manual"`]). If the list has a length greater than 1, then a multi-class model will be trained.
+#### contrasts
+- `train_validation`: list of image modalities loaded for the training and validation. If `multichannel`, then this list represent the different channels of the input tensors. Otherwise, the modalities are mixed and the model has only one input channel.
```

Comment:
```suggestion
- `train_validation`: list of image modalities loaded for the training and validation. If `multichannel=True`, this list represents the different channels of the input tensors. Otherwise, the modalities are mixed and the model only has one input channel.
```